server/crashmanager/forms.py

```diff
@@ -178,31 +178,6 @@ class Meta:
 
 
 class UserSettingsForm(ModelForm):
-    helper = FormHelper()
-    helper.layout = Layout(
-        "defaultToolsFilter",
-        Row(
-            Field("defaultProviderId", wrapper_class="col-md-6"),
-            Field("defaultTemplateId", wrapper_class="col-md-6"),
-        ),
-        "email",
-        HTML("""<p><strong>Subscribe to notifications:</strong></p>"""),
-        "inaccessible_bug",
-        "coverage_drop",
-        "bucket_hit",
-        "tasks_failed",
-        Submit("submit", "Save settings", css_class="btn btn-danger"),
-    )
-    defaultToolsFilter = ModelMultipleChoiceField(
-        queryset=Tool.objects.all(),
-        label="Select the tools you would like to include in your default views:",
-        widget=CheckboxSelectMultiple(),
-        required=False,
-    )
-    defaultProviderId = ModelChoiceField(
-        queryset=BugProvider.objects.all(), label="Default Provider:", empty_label=None
-    )
-    defaultTemplateId = ChoiceField(label="Default Template:")
```

Comment:
We removed these, in favor of passing the context directly in easily accessible format to the frontend, to avoid too many property drilling (e.g `form.initial.xxxx` on the frontend)